dandiapi/api/models/oauth.py

```diff
@@ -0,0 +1,20 @@
+import re
+from django.core.exceptions import ValidationError
+from oauth2_provider.models import AbstractApplication
+
+
+class Application(AbstractApplication):
```

Comment:
See if you can use `Application(Application)` instead to avoid the extra model/migration

Reply:
I tried this and seems a migration is unavoidable because `OAUTH2_PROVIDER_APPLICATION_MODEL` needs to be set to a custom model. I get
```
api.Application.application_ptr: (fields.E301) Field defines a relation with the model 'oauth2_provider.Application', which has been swapped out.
```

Reply:
:(
This should be renamed to something more indicative of it's purpose, also docstrings

Reply:
Done